### PR TITLE
Star rating and search components

### PR DESCRIPTION
**Pull request by melfuechec**
- Add UserRating component
- Add Search component (unfinished)
- Styling fixes
Screen Shot 2023-03-26 at 9 35 31 PM

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ function App() {
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
         <p>Welcome To Friends' Tomatoes</p>
+        <Search></Search>
```
Comment (mfuechec) [inline src/App.js:20]: <Search />

File: src/Search.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { key } from "../src/secrets";
```
Comment (mfuechec) [inline src/Search.js:2]: nice

File: src/Search.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+export default function Search() {
+  const api = `https://api.themoviedb.org/3/search/multi?api_key=${key}&language=en-US&&page=1&include_adult=true`;
+
+  const searchMulti = (setMulti) => {
+    fetch(api)
+      .then((response) => response.json())
+      .then((data) => setMulti(data));
```
Comment (mfuechec) [inline src/Search.js]: Could you come up with a better name for setMulti and searchMulti? This is an array of movie data from the api? Maybe setFilms?

File: src/UserRating.js [hunk 4]
```diff
@@ -7,7 +7,7 @@ function UserRating() {
 
   return (
     <div>
-      <Typography component="legend">TomatoMeter</Typography>
+      <Typography component="legend"></Typography>
```
Comment (mfuechec) [inline src/UserRating.js:10]: <Typography component="legend" />

File: src/utils/searchMulti.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import { key } from "../secrets";
+const api = `https://api.themoviedb.org/3/search/multi?api_key=${key}&language=en-US&&page=1&include_adult=true`;
+
+const searchMulti = (setMulti) => {
```
Comment (mfuechec) [inline src/utils/searchMulti.js]: I don't think this file is being used at all. Delete it?

**Review by mfuechec — COMMENTED** [review]
Could you clean up the description of the PR? The title should be shorter - 'Add star rating component, add search bar, styling fixes'. Convention is to always use the present tense, which feels weird, but I just think of it as finishing the sentence, 'This PR will...'
Under the title there's a spot to describe the PR. This should be a bulleted list of changes, so:
- Add UserRating component
- Add Search component (unfinished)
- Styling fixes
Then, you should use cmd-4 to take a snapshot of the website with these component and upload it in the description.

File: src/Search.js (hunk 3 above)
Comment (melfuechec) [reply src/Search.js]: changed to searchMovies for the fetch request and to setSearchInput for the state and handleChange

File: src/utils/searchMulti.js (hunk 5 above)
Comment (melfuechec) [reply src/utils/searchMulti.js]: deleted

File: src/App.js (hunk 1 above)
Comment (mfuechec) [reply src/App.js:20]: I don't know why that comment didn't show up. I was saying you could shorten this line to `<Search />`

File: src/UserRating.js (hunk 4 above)
Comment (mfuechec) [reply src/UserRating.js:10]: And this one could be `<Typography component="legend" />`

**Review by mfuechec — APPROVED** [review]
It's tight, but try to give unique commit names in the future.

**Pull request merged**